docs/en/Configuring-Skill-Instance.rst

```diff
@@ -0,0 +1,88 @@
+**************************
+Configuring Skill Instance
+**************************
+
+Skill
+=====
+
+The ``Skill`` object is the integration of all your skill logic. It is responsible for initializing SDK utilties such as the ``AttributesManager`` and ``ServiceClientFactory`` and also kick off the request handlering process.
+
+Available Methods
+-----------------
+
+.. code-block:: javascript
+
+  invoke(requestEnvelope : RequestEnvelope, context? : any) : Promise<ResponseEnvelope>;
+
+Skill Builders
+==============
+
+The ``SkillBuilder`` provides helper functions for constructing the ``Skill``, setting custom user agent and create lambda integration handler. ASK SDK v2 for Node.js provides different implementation of ``SkillBuilder`` that offer different level of customization support, which is available through ``SkillBuilders`` object.
```

Comment:
different implementations of ``SkillBuilder`` that offer different levels of customization support, which are available through